MyOnboarding/test/MyOnboardingApp.Api.Tests/TodoListControllerTest.cs

```diff
@@ -58,12 +88,27 @@ public async Task Put_IdSpecifiedTextSpecified_ReturnsOkStatusCode()
         [Test]
         public async Task Post_NewTextSpecifiedInRequestBody_ReturnsCorrectResponse()
         {
-            var msg = await _controller.GetMessageFromAction(controller => controller.PostAsync(new TodoListItem { Id = _expectedId, Text = "newText" }));
-            TodoListItem itemFromMsg;
-            msg.TryGetContentValue(out itemFromMsg);
+            var itemToAdd = new TodoListItem {Text = "newText"};
+            var expectedItem = new TodoListItem {Id = _expectedId, Text = "newText"};
+            _repository.AddNewItemAsync(itemToAdd).Returns(expectedItem);
+            const string expectedUrl = "expected/Url";
+            _itemUrlLocator.GetListItemUrl(Arg.Any<Guid>()).Returns(expectedUrl);
```

Comment:
Try to avoid `.Any<>` as much as possible - you want to be sure that `GetListItemUrl` is called with correct `id`, don't you?